Simulation/TSoG_X1_Sim.m

```diff
@@ -92,8 +99,12 @@
     Results.Time(i)  = SimData.Time;
     Results.FSM_state(i)     = SimData.Plane.FSM_state;
     Results.PitchInput(i)    = SimData.TestCase.GetPitch(SimData.Time);
-    Results.ThrottleInput(i) = SimData.TestCase.GetThrottle(SimData.Time);
-
+    Results.ThrottleInput(i) = SimData.battery_ratio*SimData.TestCase.GetThrottle(SimData.Time);
+    
+    #updates on battery for the next iteration
+    Results.battery_status(i) = Results.battery_status(i-1) - SimData.battery_update;
+    SimData.battery_ratio = Results.battery_status(i) / SimData.battery_total_cap;
```

Comment:
A LiPo battery actually goes from full charge (12.6V) to min charge (~9V). 

If we use this "battery_ratio" the thrust will reduce linearly until the battery capacity is 0. However, LiPo batteries can output up to 11.1V all the way down to 15% capacity ([Source](https://blog.ampow.com/lipo-voltage-chart/))

I have made a comment as well that we need to adjust this since after doing some further research the Max Volatage is actually 12.6V not 11.1V as previously thought.

Reply:
A thing to keep in mind is that these numbers are only valid for 3S LiPos. The usual formula is `S * 3.7V` where S is the number of batteries in series and 3.7V is the nominal voltage for a regular LiPo.
And yeah, the discharge curves of LiPos are not linear ([example](https://electronics.stackexchange.com/questions/32321/lipoly-battery-when-to-stop-draining)).